packages/ember-core/src/components/pagination.gts

```diff
@@ -246,6 +247,7 @@ export default class Pagination extends Component<PaginationSignature> {
         role={{if this.canStepBackward "button"}}
         {{! @glint-expect-error - Known Glint issue - #661 }}
         {{(if this.canStepBackward (modifier on "click" this.stepBackward))}}
+        id="page-previous"
```

Comment:
IDs are meant to be unique across an entire page - if you render multiple pagination components on the same page, you'll have multiple elements with the same ID. You can add a class if it's necessary to specify which elements you want to hide.